### PR TITLE
Refactor: Rename stage to state

### PR DESCRIPTION
**Pull request by adrianchiris**
Its all state eventually. lets make it consistent.

### CODE REVIEW

**Comment by adrianchiris** [comment]
Need to test this one on a setup

**Review by rollandf — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,32 +2,32 @@
...
 
 - [NVIDIA Network Operator](#nvidia-network-operator)
-  * [Documentation](#documentation)
-  * [Prerequisites](#prerequisites)
-    + [Kubernetes Node Feature Discovery (NFD)](#kubernetes-node-feature-discovery--nfd-)
-  * [Resource Definitions](#resource-definitions)
-    + [NICClusterPolicy CRD](#nicclusterpolicy-crd)
-      - [NICClusterPolicy spec:](#nicclusterpolicy-spec-)
-        * [Example for NICClusterPolicy resource:](#example-for-nicclusterpolicy-resource-)
+  - [Documentation](#documentation)
```
Comment (adrianchiris) [inline README.md:5]: vs code auto formatting since the entry above started with a `-`

Comment (e0ne) [reply README.md:5]: how will we handle this with future ToC updates?

Comment (adrianchiris) [reply README.md:5]: copy pasting in vscode will do the needed conversion.

LMK your preference, i can just revert this if you dont like.

Comment (e0ne) [reply README.md:5]: LGTM as we've got some tool to generate ToC in a new format

**Pull request merged**